README.md

```diff
@@ -65,3 +65,9 @@ In the repository's root directory. Eg:
 	"matrix_room_id": "!yBKstWVBkwzUkPslsp:matrix.parity.io"
 }]
 ```
+
+## CI/CD
+
+ - Deployment via gitlab is done by tagging master with `/^pre-v[0-9]+\.[0-9]+.*$/` for staging or `/^v[0-9]+\.[0-9]+.*$/` for production. 
+ - The environment variables for both staging and production live in the helm `kubernetes/processbot/values*.yml` files.
```

Comment:
Are these the only files where new env vars need to be added? What about `templates/env-secrets.yaml`, `templates/processbot-secret.yaml`? I assume new secrets need to be added to one or the other and the difference is that `env-secrets.yaml` is used for both staging and production, correct? And then there is a complete looking list of env vars in `templates/processbot.yaml`, with most values hard-coded. (Maybe this would be obvious if I knew Helm. :)